packages/jaeger-ui/src/components/TracePage/TraceGraph/TraceGraph.js

```diff
@@ -16,29 +16,23 @@
 
 import * as React from 'react';
 import { Card, Icon, Button, Tooltip } from 'antd';
+import cx from 'classnames';
 import { DirectedGraph, LayoutManager } from '@jaegertracing/plexus';
-import DRange from 'drange';
 
 import { getNodeDrawer, MODE_SERVICE, MODE_TIME, MODE_SELFTIME, HELP_TABLE } from './OpNode';
-import convPlexus from '../../../model/trace-dag/convPlexus';
-import TraceDag from '../../../model/trace-dag/TraceDag';
-
-import type { Trace, Span, KeyValuePair } from '../../../types/trace';
+import {
+  setOnEdgesContainer,
+  setOnNodesContainer,
+  setOnNode,
+} from '../../TraceDiff/TraceDiffGraph/TraceDiffGraph';
 
 import './TraceGraph.css';
 
-type SumSpan = {
-  count: number,
-  errors: number,
-  time: number,
-  percent: number,
-  selfTime: number,
-  percentSelfTime: number,
-};
-
 type Props = {
   headerHeight: number,
-  trace: Trace,
+  ev: Object,
```

Comment:
if `convPlexus`'s return value was typed, then this and other places could simply import that return value. I started trying to type it but wasn't quite satisfying flow and the errors it gave were not simple to debug. Since we're about to convert to typescript, I'd rather add a todo comment.
`// TODO: typescript conversion - use convPlexus return type`

Reply:
I was mostly keeping it similar to the existing implementation. `this.ev` -> `this.props.ev`